plugins/Live/templates/getLastVisitsStart.twig

```diff
@@ -85,7 +85,7 @@
 {{ action.serverTimePretty }}
 {% if action.timeSpentPretty is defined %}{{ 'General_TimeOnPage'|translate }}: {{ action.timeSpentPretty|raw }}{% endif %}
 {%- endset %}
-                                    <img src="plugins/Live/images/file{{ col }}.png" title="{{ action.url|e('html_attr') }}&mdash;{{- title -}}"/>
+                                    <img src="plugins/Live/images/file{{ col }}.png" title="{{ action.url|truncate(120)|e('html_attr') }}<br/>{{- title -}}"/>
```

Comment:
Not sure about the truncate, usually we don't truncate data on hover I think and show full URL.  @mattab

Reply:
Indeed it is better not to truncate to allow people to see the full value.. it is ok if the value wraps over several lines and actually a good thing 👍

Reply:
Removed.